MCNPInput.cpp

```diff
@@ -1273,16 +1296,20 @@ void InputDeck::parseSurfaces( LineExtractor& lines ){
     if( do_line_continuation( lines, token_buffer ) ){
       continue;
     }
+    mnemonic = findMnemonic( token_buffer.at(1), token_buffer.at(2) );
    
-    SurfaceCard* s = new SurfaceCard(*this, token_buffer);
+    int numFacets = mnemToNumFacets( mnemonic );
```

Comment:
You could make this `mnemToNumFacets(findMnemonic(...))` to avoid the once-used variable `mnemonic